lib/i18n/event/event_en.i18n.json

```diff
@@ -216,5 +216,9 @@
         "whatTicketPrice": "What is the price of the ticket?",
         "whatTicketTokenPrice": "What is the token price of the ticket?",
         "selectChain": "Select chain"
+    },
+    "speakers": {
+        "speakersCountInfo": "$speakersCount speakers",
```

Comment:
what's if speaker is 1 ? speakers is not correct, please search "ordinal" inside our codebase and follow the pattern